tests/test_indexing.py

```diff
@@ -718,7 +718,7 @@ class TestTakeAlongAxis:
         ],
     )
     def test_argequivalent(self, func, argfunc, kwargs):
-        a = dpnp.random.random(size=(3, 4, 5))
+        a = dpnp.asarray(numpy.random.random(size=(3, 4, 5)))
```

Comment:
Do we need to place some TODO comment here (to roll back the change once the issue with CUDA support is resolved for random)?
And also probably in some other places below.